src/shipit_frontend/src/App/CodeCoverage.elm

```diff
@@ -1,249 +0,0 @@
-module App.CodeCoverage exposing (..)
```

Comment:
There's probably some code under src/shipit_frontend that is referencing this that should be removed too.